share/messages_to_error_log.txt

```diff
@@ -11596,6 +11596,15 @@ ER_IB_SLOWRM_CREATE_DIR
 ER_IB_SLOWRM_MOVE_FILE
   eng "%s"
 
+ER_DDSE_CHANGE
+  eng "MySQL DDSE is changing from '%s' to '%s'."
+
+ER_DD_ENGINE_NOT_FOUND
+  eng "The DD_ENGINE setting for the data dictionary was not found. Starting the server using DD_ENGINE= '%u'."
```

Comment:
```suggestion
  eng "The DD_ENGINE setting for the data dictionary was not found. Starting the server using DD_ENGINE='%s'."
```

and a corresponing change in the code. IMHO the engines must be identified by their names, not by enum values

Reply:
Good point. use name is better than enum. fix both